Student.js

```diff
@@ -188,6 +188,12 @@ function std() {
 }
 
 function res() {
+for(k=0;k<100;k++){
+if(i==200)
+{
+console.logO("Res Data",k)

```

Comment:
Correct the typo in the console.log function to prevent runtime errors. [important]